indico/modules/events/requests/templates/event_request_details.html

```diff
@@ -5,10 +5,19 @@
 
 {% set has_req = req and not req.state.name == 'withdrawn' %}
 {% set show_management = is_manager and has_req %}
+{% set button_title -%}
+    {%- if not req -%}
+        {% trans %} Send request {% endtrans %}
```

Comment:
Please don't keep surrounding whitespace inside trans blocks. This is rather ugly during translation since translators need to keep them there too. If it's relevant, you could always add it at the place where you use the translated string (ie keep the space outside the trans blocks) but here it doesn't seem to be needed at all.